### PR TITLE
Update 4_rowlevelsecuirty.md rename cudos to cid

### PR DESCRIPTION
**Pull request by vmindru**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md [hunk 1]
```diff
@@ -78,17 +78,17 @@ You can select different levels of access. Tag one of the following and the use
 
 ![Images/rls_organization_accounts_tags.png](/Cost/200_Cloud_Intelligence/Images/rls/rls_organization_accounts_tags.png?classes=lab_picture_small)
 
-4. Add the Key **cudos_users** and the Value of any **emails** you wish to allow access. These are colon delimited. Once added click **Save changes**
+4. Add the Key **cid_users** and the Value of any **emails** you wish to allow access. These are colon delimited. Once added click **Save changes**
```
Comment (awssteph) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md:81]: Have you changed the code? Lets make the default in the python cudos_user

File: content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md [hunk 2]
```diff
@@ -78,17 +78,17 @@ You can select different levels of access. Tag one of the following and the use
...
 
-![Images/rls_organization_accounts_cudostags.png](/Cost/200_Cloud_Intelligence/Images/rls/rls_organization_accounts_cudostags.png?classes=lab_picture_small)
```
Comment (awssteph) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md:83]: Can you remove the old picture please?

File: content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md [hunk 3]
```diff
@@ -78,17 +78,17 @@ You can select different levels of access. Tag one of the following and the use
...
+![Images/rls_organization_accounts_cidtags.png](/Cost/200_Cloud_Intelligence/Images/rls/rls_organization_accounts_cidtags.png?classes=lab_picture_small)
 
 5. Repeat on all resources with relevant emails. 
 
 ## Deploy Lambda Function
 
 Using AWS CloudFormation we will deploy the lambda function to collect these tags. 
 
-1. Log into your account with CID. Click [Launch CloudFormation template](https://console.aws.amazon.com/cloudformation/home#/stacks/new?&templateURL=https://aws-well-architected-labs.s3-us-west-2.amazonaws.com/Cost/Labs/row-level-security/cudos_rls.yaml&stackName=CIDRowLevelSecurity)
```
Comment (awssteph) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md:91]: delete the old file please

File: content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md [hunk 4]
```diff
@@ -154,7 +154,7 @@ Once you have deployed your modules you will be able to test your Lambda functio
 
 5. The function will run, it will take a minute or two given the size of the Organizations files and processing required, then return success. Click **Details** and view the output. 
 
-6. You can go to your bucket in S3 and there should be a file in the folder CUDOS_RLS. 
+6. You can go to your bucket in S3 and there should be a file in the folder CID_RLS. 
 ![Images/rls_s3_object.png](/Cost/200_Cloud_Intelligence/Images/rls/rls_s3_object.png?classes=lab_picture_small)
```
Comment (awssteph) [inline content/Cost/200_Labs/200_Cloud_Intelligence/Customizations/Guides/4_rowlevelsecuirty.md:158]: I think you will need to update the picture for this. Make sure you hide your role, its in the same size format and you use orange boxes. See info here https://wellarchitectedlabs.com/contributing/04_updatingexisting/#picture-updates

**Comment by awsdunc** [comment]
Hi @vmindru are you still working on this PR? If so can you address the requested changes and resolve the conflicts please.

**Comment by vmindru** [comment]
will abandon this one, and pusha  a new one.

**Pull request closed without merging**